docs/INSTALL.md

```diff
@@ -50,43 +42,38 @@ FAT setup may be observed.
 ReactOS can be installed from the source distribution or from the bootable CD
 distribution. The two ways to install ReactOS are explained below.
 
-
-3.1 Installation from sources
+### 3.1 Installation from sources
 
 If you don't have an existing ReactOS installation you want to upgrade, then
 build a bootable CD as described above. Burn the CD image, boot from it, and
 follow the instructions to install ReactOS.
 
 If you have an existing ReactOS installation you want to upgrade, then to
-install ReactOS after building it, type 'ninja install'. This will create
-the directory 'reactos' in the top directory. Copy the contents of this
+install ReactOS after building it, type `ninja install`. This will create
+the directory `reactos` in the top directory. Copy the contents of this
 directory over the existing installation.
 
 If you don't want to copy the files manually every time you run a
-'ninja install', then you can specify the directory where the files are
+`ninja install`, then you can specify the directory where the files are
 to be copied to during installation.
 
 Set the ROS_INSTALL environment variable. If you are on Windows this could be
 done by:
 
-  set ROS_INSTALL=c:\reactos
+`set ROS_INSTALL=c:\reactos`
 
 If you are on Linux this could be done by:
 
-  export ROS_INSTALL=/mnt/windows/reactos
+`export ROS_INSTALL=/mnt/windows/reactos`
 
 Now run 'ninja install' to install the files to the new location.
 
 
```

Comment:
```suggestion
```
I would however suggest we keep the double new line before a section (not the subsections) starts.

Reply:
I think I reformatted it how you wanted it, let me know if I didn't. Don't care too much about the specific format as long as it's consistent and easy to read.